webrtc-stats.html

```diff
@@ -1428,7 +1428,9 @@
               </dt>
               <dd>
                 <p>
-                  Only valid for video. It represents the nominal FPS value.
+                  Only valid for video. It represents the nominal FPS value before the degradation
+                  preference is applied. For sending tracks it is the current captured FPS
+                  and for the receiving tracks it is the current decoding framerate.
```

Comment:
Elsewhere on the internet (Apple, states: https://developer.apple.com/reference/avfoundation/avassettrack/1386182-nominalframerate?language=objc): 
> The nominal frame rate indicates the number of frames per second for tracks that carry a full frame per media sample